packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -784,8 +784,13 @@ export async function launch({
     }
   }
 
+  function shutdown() {
+    return controller.shutdown();
+  }
+
   return {
     blockingSend,
+    shutdown,
```

Comment:
Unfortunately until split brains I do not believe it's possible for `chain-main.js` to actually call this.